test/config_parser_SUITE.erl

```diff
@@ -2557,29 +2557,28 @@ mod_revproxy(_Config) ->
           ]).
 
 mod_roster(_Config) ->
-    Riak = #{<<"bucket_type">> => <<"rosters">>,
-             <<"version_bucket_type">> => <<"roster_versions">>},
-    Base = #{<<"iqdisc">> => #{<<"type">> => <<"one_queue">>},
-             <<"versioning">> => false,
-             <<"store_current_id">> => false,
-             <<"backend">> => <<"mnesia">>,
-             <<"riak">> => Riak},
-    MBase = [{iqdisc, one_queue},
-             {versioning, false},
-             {store_current_id, false},
-             {backend, mnesia},
-             {bucket_type, <<"rosters">>},
-             {version_bucket_type, <<"roster_versions">>}],
     T = fun(Opts) -> #{<<"modules">> => #{<<"mod_roster">> => Opts}} end,
-    run_multi([
-            ?_eqf(modopts(mod_roster, lists:sort(MBase)), T(Base)),
-            ?_errf(T(#{<<"versioning">> => 1})),
-            ?_errf(T(#{<<"store_current_id">> => 1})),
-            ?_errf(T(#{<<"backend">> => 1})),
-            ?_errf(T(#{<<"backend">> => <<"iloveyou">>})),
-            ?_errf(T(#{<<"riak">> => #{<<"version_bucket_type">> => 1}})),
-            ?_errf(T(#{<<"riak">> => #{<<"bucket_type">> => 1}}))
-          ]),
+    M = fun(Cfg) -> modopts(mod_roster, Cfg) end,
+
+    ?eqf(M([{iqdisc, one_queue}]),
+       T(#{<<"iqdisc">> => #{<<"type">> => <<"one_queue">>}})),
```

Comment:
Do we need it if we already do `check_iqdisc`?

Reply:
True, I'll have that removed and more `check_iqdisc` to the top as it's the first option in the doc.